src/OpenLoco/S5/S5.cpp

```diff
@@ -371,6 +371,7 @@ namespace OpenLoco::S5
         catch (const std::exception& e)
         {
             std::fprintf(stderr, "Unable to save S5: %s\n", e.what());
+            Ui::showMessageBox("Unable to save S5", e.what());
```

Comment:
I think we can use an in-game error window for any S5 related loading errors. @duncanspumpkin, any thoughts?

Reply:
Hmm not too sure. What would happen if title screen failed to load? I guess that doesn't mean the ui has failed to load so could present an error but not too sure.

Reply:
If the title screen fails to load, you'll still get a game window:

![Screenshot (3)](https://user-images.githubusercontent.com/604665/130223307-8e7af875-aeb0-4c21-880b-6bbec61a4d1f.png)